guis/desktop/gui.py

```diff
@@ -166,7 +167,7 @@ def create_crs_info(self):
         crs_info_hsizer = wx.BoxSizer(wx.VERTICAL)
         # Input text
         self.crs_info_text = wx.StaticText(self.panel, -1, "",
-                                           style=wx.ALIGN_LEFT, size=(600, 300))
+                style=wx.ALIGN_LEFT, size=(400, 300))
```

Comment:
Let's pick one consistent style for indentation within parentheses. In the core, I indented to the opening parentheses, which takes up more horizontal space and may need more vertical space, but looks good (?).